pkg/util/runtimeconfig/manager.go

```diff
@@ -44,20 +45,26 @@ type Manager struct {
 	config    interface{}
 
 	configLoadSuccess prometheus.Gauge
+	configInfo        *prometheus.GaugeVec
 }
 
 // NewRuntimeConfigManager creates an instance of Manager and starts reload config loop based on config
 func NewRuntimeConfigManager(cfg ManagerConfig, registerer prometheus.Registerer) (*Manager, error) {
 	mgr := Manager{
 		cfg: cfg,
 		configLoadSuccess: prometheus.NewGauge(prometheus.GaugeOpts{
-			Name: "cortex_overrides_last_reload_successful",
-			Help: "Whether the last config reload attempt was successful.",
+			Name: "cortex_runtime_config_last_reload_successful",
+			Help: "Whether the last runtime-config reload attempt was successful.",
 		}),
+		configInfo: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Name: "cortex_runtime_config_info",
```

Comment:
`_config_info` sounds too generic to me. I would suggest calling it `cortex_runtime_config_hash` with `sha256` label (instead of prefix). Same for metric in main package. (This is just a personal preference though)

Reply:
My thinking was that maybe in the future we also want to add more labels, like filepath to the metric, so I have kept it more generic.

I was not too sure if this use case is cover by (https://prometheus.io/docs/practices/naming/):

> foobar_build_info (for a pseudo-metric that provides metadata about the running binary)

Reply:
> maybe in the future we also want to add more labels, like filepath to the metric

We can still add this info in the labels. I agree with Peter about naming it `cortex_runtime_config_hash`

Reply:
Renamed it in the latest iteration